pom.xml

```
@@ -124,6 +132,39 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.2.0</version>
```

```suggestion
                <version>3.4.1</version>
```

This is the most recent version: https://search.maven.org/artifact/org.apache.maven.plugins/maven-javadoc-plugin

---

You might also want to use `${maven-javadoc-plugin.version}`